api/src/main/kotlin/com/openlattice/collaborations/CollaborationsApi.kt

```diff
@@ -23,6 +24,7 @@ interface CollaborationsApi {
         const val COLLABORATION_ID_PATH = "/{$COLLABORATION_ID_PARAM}"
         const val DATA_SET_ID_PARAM = "dataSetId"
         const val DATA_SET_ID_PATH = "/{$DATA_SET_ID_PARAM}"
+        const val ID_PARAM = "id"
```

Comment:
```suggestion
        const val IDS_PARAM = "ids"
```

Reply:
the param should be id. if multiple ids are passed, they're each assigned their own in repeat
`?id=1234&id=4567`

Reply:
We discussed this in slack, agreeing to keep it as `id`.